src/MailChannel.php

```diff
@@ -29,9 +39,7 @@ class MailChannel implements ChannelInterface
      */
     public function isHandling(MessageInterface $message, RecipientInterface $recipient)
     {
-        return isset($recipient->mail_to)
-            && isset($message->mail_subject)
-            && isset($message->mail_body);
+        return ($this->getMessageBag($message) && $this->getRecipientBag($recipient));
```

Comment:
Remove the parenthesis. Maybe do ! `is_null` checks when the bags return `null`.